std_msgs/Header header
LandmarkDetection[] landmarks
